src/shared/types.ts

```diff
@@ -21,4 +21,11 @@ export interface INodePool {
   status: INodePoolStatus;
 }
 
+export interface IKeyPair {
```

Comment:
This location is temporary, until we switch to a more domain-oriented folder structure